resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -1,115 +1,370 @@
+        }
+      }
+    },
+    {
+      "Sid": "CreateSnapshot",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateSnapshot"
+      ],
+      "Resource": "arn:aws:ec2:*:*:volume/*"
+    },
+    {
+      "Sid": "CopySnapshot",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CopySnapshot"
+      ],
+      "Resource": "arn:aws:ec2:*:*:snapshot/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:RequestTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "GetSerialConsoleAccessStatus",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:GetSerialConsoleAccessStatus"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "SendSerialConsoleSSHPublicKey",
+      "Effect": "Allow",
+      "Action": [
+        "ec2-instance-connect:SendSerialConsoleSSHPublicKey"
+      ],
+      "Resource": "arn:aws:ec2:*:*:instance/*"
+    },
+    {
+      "Sid": "CreateNetworkInsightsPathTagRestriction",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateNetworkInsightsPath"
+      ],
+      "Resource": "arn:aws:ec2:*:*:network-insights-path/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:RequestTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "CreateNetworkInsightsPath",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateNetworkInsightsPath"
+      ],
+      "Resource": [
+        "arn:aws:ec2:*:*:internet-gateway/*",
+        "arn:aws:ec2:*:*:instance/*",
+        "arn:aws:ec2:*:*:network-interface/*",
+        "arn:aws:ec2:*:*:transit-gateway/*",
+        "arn:aws:ec2:*:*:vpc-endpoint/*",
+        "arn:aws:ec2:*:*:vpc-endpoint-service/*",
+        "arn:aws:ec2:*:*:vpc-peering-connection/*",
+        "arn:aws:ec2:*:*:vpn-gateway/*"
+      ]
+    },
+    {
+      "Sid": "DescribeNetworkInsightsPath",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeNetworkInsightsPaths",
+        "ec2:DescribeNetworkInsightsAnalyses"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DeleteNetworkInsightsPath",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DeleteNetworkInsightsPath"
+      ],
+      "Resource": "arn:aws:ec2:*:*:network-insights-path/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "StartNetworkInsightsAnalysisTagRestriction",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:StartNetworkInsightsAnalysis"
+      ],
+      "Resource": [
+        "arn:aws:ec2:*:*:network-insights-analysis/*"
+      ],
+      "Condition": {
+        "StringEquals": {
+          "aws:RequestTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "StartNetworkInsightsAnalysis",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:StartNetworkInsightsAnalysis"
+      ],
+      "Resource": [
+        "arn:aws:ec2:*:*:network-insights-path/*"
```

Comment:
What's different between this and line 158?

Reply:
this command analyze the path created above, so in order to use reachability analyzer, we need this action 
([link](https://docs.aws.amazon.com/cli/latest/reference/ec2/start-network-insights-analysis.html))

Reply:
the resources in the 2 are different: in line 158 is "arn:aws:ec2:*:*:network-insights-analysis/*" with "aws:RequestTag/red-hat-managed": "true" and this is "arn:aws:ec2:*:*:network-insights-path/*" with "aws:ResourceTag/red-hat-managed": "true"